app/assets/javascripts/visualizations/highvis/bar.coffee

```diff
@@ -103,6 +103,30 @@ $ ->
             [fieldTitle(data.fields[fid]), groupedData[fid][gid]]
 
           @chart.addSeries options, false
+
+          # Draw error bars if that analysis type is selected
+          if @configs.analysisType == @ANALYSISTYPE_MEAN_ERROR
+            errors =
+              type: 'errorbar'
+              name: "Error for #{data.groups[gid]}" or data.noField()
+
+            errors.data = for fid in data.normalFields when fid in fieldSelection
+              barData = data.selector fid, gid
+              if barData.length == 1
+                []
+              else
+                mean = groupedData[fid][gid]
+                innerStd = barData.map((x) -> (x - mean) ** 2).reduce((x, y) -> x + y)
+                stdDev = Math.sqrt((1 / (barData.length - 1)) * innerStd)
+                if globals.configs.logY != true
+                  [mean - stdDev, mean + stdDev]
+                else if mean > 0
+                  [mean, mean + stdDev]
+                else
+                  []
```

Comment:
Are negative means not useful? It seems to me that if mean >= 0 then [mean, mean + stdDev] else [mean, mean - stdDev]. Admittedly, this is just intuition, and not understanding. I would appreciate it if you would clarify this for me.